### ISSUE TITLE
Inputs fail to validate

### ISSUE THREAD

**Issue by smunteankestra** [kestra-io/kestra#5854]
### Describe the issue

Related to a previous issue https://github.com/kestra-io/kestra/issues/5731

https://www.loom.com/share/1f451ab718b64022b3c88c4c5c82a761?sid=314afd52-e66e-48dd-9a1e-f3004f4f24ff

Flow:

id: autotest
namespace: autotest
inputs:
  - id: test
    type: STRING
  - id: bool
    type: BOOLEAN
#  - id: depend
#    type: STRING
tasks:
  - id: hello
    type: io.kestra.plugin.core.log.Log
    message: "Hello World! {{ inputs.test }}"
    
    
    
Also can be reproduced by running the  automated test ""inputFormFlow()      

### Environment

- Kestra Version: develop

**Comment by anna-geller** [comment]
I think we misunderstood each other, the video shows that everything is now working :) 

The only issue that might not working be fully working yet is when you specify UNDEFINED in the boolean, because the Executions Overview captures it as `false`

**Comment by smunteankestra** [comment]
this is what I get on my side - but maybe version is not the latest 
 this is the latest I could pull
https://github.com/user-attachments/assets/087841fb-311a-4508-8476-aa9e375ee47c

**Comment by anna-geller** [comment]
good catch, this happens only on Kafka backend https://share.descript.com/view/5MyN65Inw9e 

I would transfer to EE repo but GitHub doesn't let me

**Comment by smunteankestra** [comment]
@anna-geller I reproduced this on posgres BE

**Comment by anna-geller** [comment]
thx for the info, definitely something that happens only on EE preview, maybe still a build issue then? on OSS preview, I could no longer reproduce

**Comment by smunteankestra** [comment]
yes, I can confirm on OSS looks good - but EE has issues

**Comment by anna-geller** [comment]
EE build is outdated. Loic will have a look and once we fix the EE build, we'll likely close this issue 👍
